broker-core/src/main/java/io/zeebe/broker/engine/impl/DeploymentDistributorImpl.java

```diff
@@ -184,12 +207,42 @@ private void pushDeploymentToPartition(final int partitionLeaderId, final int pa
                 }));
   }
 
-  private void handleResponse(byte[] response, int partitionLeaderId, int partition) {
+  private void createResponseSubscription(
```

Comment:
We are re-creating the subscription on retry does this make sense? Is this problematic ?

Reply:
I couldn't find any documentation regarding that so to be safe I added that check to only create a subscription if we don't find one with that topic (line 212).

Reply:
Ah sorry have overseen that. Might make sense to just open it at 
`pushDeployment(
      final long key, final long position, final DirectBuffer buffer)` ?

Reply:
There are some paths were the don't push the deployment (like if there is only one partition) and then we have this permanently open subscription I think. I'd have to add additional checks to make sure that didn't happen

Reply:
Ok but on `deployOnMultiplePartitions` it will work?

Reply:
Sure, I moved it now